packages/paste-core/components/combobox/src/Combobox.tsx

```diff
@@ -26,21 +34,129 @@ const StyledInputAsSelect = styled(InputElement)(
 );
 /* eslint-enable */
 
-export interface ComboboxProps extends Omit<FormInputProps, 'id' | 'type'> {
+type Item = string | {};
+
+export interface ComboboxProps extends Omit<FormInputProps, 'id' | 'type' | 'value'> {
   autocomplete?: boolean;
   helpText?: string | React.ReactNode;
-  initialSelectedItem?: UseComboboxPrimitiveProps<any>['initialSelectedItem'];
-  items: UseComboboxPrimitiveProps<any>['items'];
-  itemToString?: UseComboboxPrimitiveProps<any>['itemToString'];
+  initialIsOpen?: UseComboboxPrimitiveProps<Item>['initialIsOpen'];
+  initialSelectedItem?: UseComboboxPrimitiveProps<Item>['initialSelectedItem'];
+  items: UseComboboxPrimitiveProps<Item>['items'];
+  itemToString?: UseComboboxPrimitiveProps<Item>['itemToString'];
   labelText: string | NonNullable<React.ReactNode>;
-  onHighlightedIndexChange?: UseComboboxPrimitiveProps<any>['onHighlightedIndexChange'];
-  onInputValueChange?: UseComboboxPrimitiveProps<any>['onInputValueChange'];
-  onIsOpenChange?: UseComboboxPrimitiveProps<any>['onIsOpenChange'];
-  onSelectedItemChange?: UseComboboxPrimitiveProps<any>['onSelectedItemChange'];
+  onHighlightedIndexChange?: UseComboboxPrimitiveProps<Item>['onHighlightedIndexChange'];
+  onInputValueChange?: UseComboboxPrimitiveProps<Item>['onInputValueChange'];
+  onIsOpenChange?: UseComboboxPrimitiveProps<Item>['onIsOpenChange'];
+  onSelectedItemChange?: UseComboboxPrimitiveProps<Item>['onSelectedItemChange'];
   optionTemplate?: (item: string | {}) => React.ReactNode;
+  selectedItem?: UseComboboxPrimitiveProps<Item>['selectedItem'];
+  inputValue?: UseComboboxPrimitiveProps<Item>['inputValue'];
+  groupItemsBy?: string;
+}
+
+interface RenderItemProps {
+  item: Item;
+  index: number;
+  getItemProps: any;
```

Comment:
This is something that we don't get the types for from Downshift and we don't even use it so i'm happy with an any here